src/ck3/data/messages.rs

```diff
@@ -62,8 +62,32 @@ impl DbKind for MessageFilterType {
         let loca = format!("message_filter_{key}");
         data.verify_exists_implied(Item::Localization, &loca, key);
 
-        vd.field_choice("display", &["feed", "toast", "hidden"]);
+        vd.field_choice("display", &["feed", "toast", "hidden", "popup"]);
         vd.field_bool("always_show");
         vd.field_bool("auto_pause");
+        vd.field_integer("sort_order");
+        vd.field_item("group", Item::MessageFilterGroupType);
+    }
+}
+
+
+#[derive(Clone, Debug)]
+pub struct MessageFilterGroupType {}
+
+inventory::submit! {
+    ItemLoader::Normal(GameFlags::Ck3, Item::MessageFilterGroupType, MessageFilterGroupType::add)
```

Comment:
When you add a new `Item` type, you have to add it in `src/items.rs` as well. In two places: once in the `Item` enum (with the other ck3 items), and once in the `path` function.

Reply:
Removed from file